packages/itwinui-css/src/menu/menu.scss

```diff
@@ -100,6 +100,26 @@ $iui-active-outline: thin solid var(--iui-color-border-accent);
   }
 
   &.iui-active {
+    @supports (selector(.iui-active:has(+ .iui-active))) {
+      // Styles active items between the first & last
+      + .iui-active {
+        border-radius: 0;
+      }
+
+      // Styles first active item with an active item following it
+      &:has(+ .iui-active) {
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+      }
+
+      // Styles last active item
+      &:has(+ :not(.iui-active)),
+      &:last-child {
+        border-bottom-left-radius: var(--iui-border-radius-1);
+        border-bottom-right-radius: var(--iui-border-radius-1);
```

Comment:
Instead of initially resetting all the border radius on [L105-107](https://github.com/iTwin/iTwinUI/pull/1142/files#diff-a340fb8a57004b5349e2d15e6cce890a749c4afd443d3323981302b08a2a77b0R105-R107) and readding it for this one case, would it be safer code to not reset the border radius but only remove the border radius in certain cases?

* first item: remove only bottom border radii
* middle items: remove top and bottom border radii
* last item: remove top border radii

This is just a suggestion. Because for example, if we ever change the border radius on [L17](https://github.com/iTwin/iTwinUI/blob/47d994cf3d58119c3d63efe60f909733c7e5d7a9/packages/itwinui-css/src/menu/menu.scss#L17) but forget to change it here, then there could be inconsistencies in the border radius specification.

Reply:
I knew there was a better way to do this, I just couldn't piece it together.  Suggestions here are much better, changes made!